lib/module.js

```diff
@@ -1,11 +1,11 @@
-module.exports = async function NuxtLoadingScreen() {
+const LoadingUI = require('./loading')
```

Comment:
Lazy require was intentional to make startup faster.

Reply:
Will revert, thanks for pointing it